docs/development-and-gardener-setup.md

```diff
@@ -139,7 +91,7 @@ SUBJECT_NAME=kubemgmt NAMESPACE=$SHOOT_NS CA=ca.pem ./contrib/gardener/scripts/c
 Install resources on the shoot:
 ```
 export KUBECONFIG=$KUBECONFIG_SHOOT
-kubectl apply -f manifests/namespace.yml
+kubectl apply -f install/charts/templates/namespace.yml
```

Comment:
Same problem here. Can a template used for creation?

Reply:
Changed the description to work with templates.